rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2292,6 +2295,20 @@ protected Coin getAmountSentToAddress(BtcTransaction btcTx, Address btcAddress)
         return v;
     }
 
+   // This method will be used by registerBtcTransfer to save all the data required on storage (utxos, btcTxHash-derivationHash),
```

Comment:
```suggestion
   // This method will be used by registerFastBridgeBtcTransaction to save all the data required on storage (utxos, btcTxHash-derivationHash),
```

I think the comment could be completely deleted